README.md

```diff
@@ -24,8 +24,8 @@ the _API integration_ page.
 ```javascript
 notifyClient
 	.sendSms(templateId, phoneNumber, personalisation, reference)
-	.then((response) => { })
-	.catch((err) => {})
+	.then(response => console.log)
```

Comment:
my js-fu may be failing - but shouldn't this just be `.then(console.log)` or `.then(response => console.log(response)`?